config/default.yaml

```diff
@@ -2,6 +2,10 @@ server:
   port: 5999
   appRoot: "/bucky"
 
+handlers:
```

Comment:
Thats the required format 

Here is how you would make your static file handler return that required response header value:
handlers:
- url: /images
  static_dir: static/images
  http_headers:
    Access-Control-Allow-Origin: http://mygame.appspot.com